### PR TITLE
Issues/#116 watch reservation list

### PR DESCRIPTION
**Pull request by nikola-mladenovic**
TestFlight build coming right up, but you can also see the video I made for the UI reference.

### CODE REVIEW

File: Sunrise Watch Extension/InterfaceModels/ReservationsInterfaceModel.swift [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+    func productImageUrl(at row: Int) -> String {
+        return reservations[row].lineItems?.first?.variant?.images?.first?.url ?? ""
+    }
+
+    // MARK: - Reservations retrieval
+
+    private func retrieveReservations() {
+        
+
```
Comment (cneijenhuis) [inline Sunrise Watch Extension/InterfaceModels/ReservationsInterfaceModel.swift]: Whitespace

**Review by cneijenhuis — APPROVED** [review]
In the video, the table view seems to have a "scroll" indicator, but it's mostly overlapped by the table cells (just popping up between them from time to time). We should either remove it or give it a different color - WDYT?

Code LGTM 👍

**Comment by nikola-mladenovic** [comment]
img_3323

This is a screenshot from the default watch Weather app - the scroll indicator is above the content (cells), it's a system one, and it has a different color depending on whether you use your finger to scroll or the digital crown.

File: Sunrise Watch Extension/InterfaceModels/ReservationsInterfaceModel.swift (hunk 1 above)
Comment (nikola-mladenovic) [reply Sunrise Watch Extension/InterfaceModels/ReservationsInterfaceModel.swift]: Good catch, removed 👍

**Comment by cneijenhuis** [comment]
Well, the white color doesn't work well with our bright cells/white background images 😢

I'm fine with merging it, but my guess is that someone else will complain as well 😉

**Comment by nikola-mladenovic** [comment]
@mmoelli - what do you think about the background color of the cells from our video? Do we want to make any changes now or leave for later?

**Comment by mmoelli** [comment]
From my point of view not critical. I would leave it as it has only as information character.  (personal opinion, I never use scroll bars directly)

**Pull request merged**